lib/themes/dosomething/paraneue_dosomething/scss/content/_user.scss

```diff
@@ -3,9 +3,21 @@
 // ----------------
 
 .profile {
+  
   .tile {
     .__tagline { display: none; }
   }
+  
+  .figure__title,  .figure__description{
+    height: 2em;
```

Comment:
Can you make this `min-height` so we don't accidentally cut off content in case of a longer description?